src/Sulu/Page/Application/Mapper/PageContentMapper.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Page\Application\Mapper;
+
+use Sulu\Page\Domain\Model\PageInterface;
+use Sulu\Bundle\ContentBundle\Content\Application\ContentPersister\ContentPersisterInterface;
+use Webmozart\Assert\Assert;
+
+/**
+ * @experimental
+ *
+ * @internal This class should be instantiated inside a project.
+ *           Use the message to create or modify an Page.
+ *           Or the inject all the mappers into a custom service.
+ *           Create an own Mapper to extend the mapper with
+ *           custom logic.
+ */
+final class PageContentMapper implements PageMapperInterface
+{
+    private ContentPersisterInterface $contentPersister;
+
+    public function __construct(ContentPersisterInterface $contentPersister)
```

Comment:
Just a friendly reminder that we are now using php8 and you can use constructor property promotion to avoid duplicate code.